libs/etterlatte-behandling-model/src/main/kotlin/no/nav/etterlatte/libs/common/generellbehandling/GenerellBehandling.kt

```diff
@@ -6,6 +6,19 @@ import java.util.*
 data class GenerellBehandling(
     val id: UUID,
     val sakId: Long,
-    val innhold: Innhold,
-    val opprettet: Tidspunkt
-)
\ No newline at end of file
+    val opprettet: Tidspunkt,
+    val type: GenerellBehandlingType,
+    val innhold: Innhold? = null
```

Comment:
```suggestion
    val innhold: Innhold?
```
tenker det kan værne oss mot noen bugs å påkreve at innhold settes eksplisitt, selv når det er null

Reply:
Antar at denne utgår nå med disse endringene? https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2177/commits/2ad076820a8367d7b98f6661229f78a9c6c7dd02

Reply:
Trenger at den er null ved opprettelse basert på type

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2177/commits/9066cc01f6d8abaf57b5ea7956997f7ee03910e7